delivery_readmes/DELIVERY.README.RNASeq.txt

```diff
@@ -0,0 +1,119 @@
+README
+=======
+
+The README describes the content of the delivered data.
+-----------------------------------------------------------------
```

Comment:
In fact I'm not really sure what this header is saying? Maybe remove the line under it and have an introduction sentence...

```suggestion
Many thanks for sequencing your samples with the SciLifeLab National Genomics Infrastructure!
This README describes the different files present in your data delivery, including information about verifying file integrity and finding results.
If you have any questions, please don't hesitate to get in touch: support@ngisweden.se
```